sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1295,15 +1295,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
-  val PANDAS_GROUPED_MAP_ASSIGN_COLUMNS_BY_POSITION =
-    buildConf("spark.sql.execution.pandas.groupedMap.assignColumnsByPosition")
+  val PANDAS_GROUPED_MAP_ASSIGN_COLUMNS_BY_NAME =
+    buildConf("spark.sql.legacy.execution.pandas.groupedMap.assignColumnsByName")
```

Comment:
Out of curiosity, wouldn't we better leave this `false` by default (which means we disable a legacy behvaiour by default)?

Reply:
Actually, we are not following this in the other SQLConf. I think either is fine. We will remove these SQLConf in Spark 3.0 release.

Reply:
Yup, okay. Was just wondering.